src/inventory_ui.h

```diff
@@ -52,6 +52,14 @@ struct inventory_entry_drawn_info {
     int text_x_start;
     int text_x_end;
     int y;
+    bool include_point( point p )const {
+        if( text_x_start <= p.x &&
```

Comment:
Would you please just put this all on one line?  General CDDA practice is to put multiple conditions on a single line if the total line length is less than 100 characters.

Reply:
hah? but I saw many cases of mutiple lined if conditions